bundle/src/main/java/com/adobe/acs/tools/csv_asset_importer/impl/CsvAssetImporterServlet.java

```diff
@@ -89,8 +89,30 @@ protected final void doPost(final SlingHttpServletRequest request, final SlingHt
         final Parameters params = new Parameters(request);
         if (params.getFile() != null) {
 
-            final long start = System.currentTimeMillis();
-            final Iterator<String[]> rows = this.getRowsFromCsv(params);
+        	Iterator<String[]> rows = null;
+			final long start = System.currentTimeMillis();
+			String inFileMimeType = params.getInFileMimeType();
+			final String tempDir = System.getProperty("java.io.tmpdir") + start;
```

Comment:
Also, as far as I can see, the temp directory is never removed. That's very problematic.